### ISSUE TITLE
A follow-up Should be Added - 1539.Kth Missing Positive Number

### ISSUE THREAD

**Issue by CarlCzerny837** [LeetCode-Feedback/LeetCode-Feedback#7440]
Your LeetCode username
v5qYY4Q65w

Category of the bug
- [√] Question
- [ ] Solution
- [ ] Language
- [ ] Missing Test Cases 

Description of the bug
This problem is same as the problem 1060.Missing Element in Sorted Array, and there is a follow-up requirement in the problem 1060, so the same follow-up requirement should be added to this problem:  Can you find a logarithmic time complexity (i.e., O(log(n))) solution?

Code you used for Submit/Run operation

The time complexity of the code below is O(n):
```
class Solution:
    def findKthPositive(self, arr: List[int], k: int) -> int:

        st = set(arr)
        cnt = 0
        p = 1
        while True:
            if p not in st:
                cnt+=1
                if cnt==k:
                    return p
            p+=1
```

The time complexity of the code below is O(log(n)):
```
class Solution:
    def findKthPositive(self, arr: List[int], k: int) -> int:

        if arr[0]>k:
            return k
        
        l = 0
        r = len(arr)
        while l < r:
            m=(l+r)//2
            x = arr[m] if m<len(arr) else 10**9
            if x-m>=k+1:
                r=m
            else:
                l =m+1

        return k-(arr[l-1]-(l-1)-1)+arr[l-1]
```

Language used for code
Python3

Expected behavior
A follow-up requirement should be added:  
**Can you find a logarithmic time complexity (i.e., O(log(n))) solution?**

**Comment by LC-Pam** [comment]
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by LC-Pam** [comment]
Thank you for your time. We've used your feedback to update the problem. 
(The log(n) solution although very neat is not "easy". Adding that as follow-up will change the difficulty of the problem.
So, we did not add the follow up here. However, we might add a similar problem with large constraints in future, so that only the log(n) solution passes. 
We also updated the statement.)
We appreciate your support!
